packages/documentation-site/patternfly-docs/content/design-guidelines/styles/colors-for-charts/colors-grid.js

```diff
@@ -1,409 +1,83 @@
 import React from 'react';
 import { Grid, GridItem } from '@patternfly/react-core';
+import '@patternfly/patternfly/patternfly-charts-theme-dark.css';
```

Comment:
Just a nit, but should we move the base charts CSS import into this file, too, just for organization? Or does it need to be in the other file? https://github.com/patternfly/patternfly-org/blob/d93dc163789d545df723003a5e5604b4582971e2/packages/documentation-site/patternfly-docs/content/design-guidelines/styles/colors-for-charts/colors-for-charts.md?plain=1#L14

```suggestion
import '@patternfly/patternfly/patternfly-charts.css';
import '@patternfly/patternfly/patternfly-charts-theme-dark.css';
```